packages/api-plugin-fulfillment-method-shipping-flat-rate/src/util/locationDenyCheck.js

```diff
@@ -0,0 +1,27 @@
+import isDestinationRestricted from "./isDestinationRestricted.js";
```

Comment:
Needs tests

Reply:
Tests are present in `filterShippingMethods.test.js`